doc/source/fmpq_poly.rst

```diff
@@ -1027,6 +1027,16 @@ Derivative and integral
 
     Sets ``res`` to the derivative of ``poly``.
 
+.. function:: void _fmpq_poly_nth_derivative(fmpz * rpoly, fmpz_t rden, const fmpz * poly, const fmpz_t den, ulong n, slong len)
+
+    Sets ``(rpoly, rden, len - n)`` to the nth derivative of 
+    ``(poly, den, len)``.  Does nothing if ``len <= n``.  
+    Supports aliasing between the two polynomials.
```

Comment:
I believe FLINT by standard supports aliasing. This could confuse the reader by implying that only this function supports aliasing.

Reply:
I copied the layout of the ```fmpq_poly_derivative``` doc

Reply:
Yeah, that's reasonable. Let's hear what the others say.

Reply:
It's good to mention it when it is an underscore function, since not all of those support aliasing (in fact most probably don't).